contracts/Arbitrum_SpokePool.sol

```diff
@@ -1,5 +1,8 @@
 // SPDX-License-Identifier: BUSL-1.1
-pragma solidity ^0.8.0;
+
+// Arbitrum only supports v0.8.19
+// See https://docs.arbitrum.io/for-devs/concepts/differences-between-arbitrum-ethereum/solidity-support#differences-from-solidity-on-ethereum
+pragma solidity 0.8.19;
```

Comment:
OOC how was this running before? Prod has the spokepool set at `^0.8.0`.

cc @nicholaspai

Reply:
I think we kinda recently changed the compiler version within the `hardhat.config.ts` file